src/cmds/scripts/pbs_postinstall.in

```diff
@@ -700,8 +700,17 @@ create_home() {
 				sed '/^[[:space:]]*resources:/ s/\"$/, vntype\"/' $sconfig > $tmp_config
 				mv $tmp_config $sconfig
 			fi
+
+			grep '^[[:space:]]*resources:.*hbmem' $sconfig > /dev/null
+			if [ $? -ne 0 ]; then
+				echo "*** Added hbmem to sched_config resources"
+				tmp_config="/tmp/config.$$"
+				sed '/^[[:space:]]*resources:/ s/\"$/, hbmem\"/' $sconfig > $tmp_config
```

Comment:
can we use --in-place option instead of doing this in two steps?

Reply:
Done!